liri.js

```
@@ -0,0 +1,155 @@
+  }
+}
+
+//my-tweets
+function tweets() {
+  let client = new twitter({
+    consumer_key: keys.twitterKeys.consumer_key,
+    consumer_secret: keys.twitterKeys.consumer_secret,
+    access_token_key: keys.twitterKeys.access_token_key,
+    access_token_secret: keys.twitterKeys.access_token_secret
+  });
+  let params = {
+    screen_name: 'djlowes',
+    count: 20
+  };
+  client.get('statuses/user_timeline', params, function(error, tweets, response) {
+    if (!error && response.statusCode === 200) {
+      for (let i = 0; i < tweets.length; i++) {
+        console.log(tweets[i].user.screen_name + ": " + tweets[i].text);
+        log(tweets[i].user.screen_name + ": " + tweets[i].text);
+      }
+    } else {
+      console.log("Tweets failed to load, this is the error: " + error);
+      log("Tweets failed to load, this is the error: " + error);
+    }
+  });
+}
+
+
+// spotify-this-song
+function spotify() {
+  let spot = new Spotify({
+    id: keys.spotifyKeys.client_ID,
+    secret: keys.spotifyKeys.client_Secret
+  });
+  if (!query) {
+    query = "The Sign"
+  }
+  let params = {
+    type: 'track',
+    query: query
+  };
+  spot.search(params, function(error, response) {
+    if (error) {
+      return console.log('Error occurred: ' + error);
+    } else {
+      // console.log(response.tracks.items[0]);
+      console.log("Artist: " + response.tracks.items[0].album.artists[0].name);
+      log("Artist: " + response.tracks.items[0].album.artists[0].name);
+      console.log("Song: " + response.tracks.items[0].name);
+      log("Song: " + response.tracks.items[0].name);
+      console.log("Preview Link: " + response.tracks.items[0].preview_url);
+      log("Preview Link: " + response.tracks.items[0].preview_url);
+      console.log("Album: " + response.tracks.items[0].album.name);
+      log("Album: " + response.tracks.items[0].album.name);
+    }
+  });
+}
+
+
+// movie-this
+function imdb() {
+  if (!query) {
+    query = 'Mr. Nobody.'
+  }
```

Instead of having a default you can maybe throw something at the user to have them give you a proper input.